addons/project/models/project.py

```diff
@@ -1333,6 +1344,16 @@ def _compute_dependent_tasks_count(self):
             for task in tasks_with_dependency:
                 task.dependent_tasks_count = dependent_tasks_count_dict.get(task.id, 0)
 
+    @api.depends('depend_on_ids', 'depend_on_ids.stage_id', 'depend_on_ids.stage_id.fold')
+    def _is_blocked(self):
+        for task in self:
+            task.is_blocked = any(not blocking_task.stage_id.fold for blocking_task in task.depend_on_ids)
```

Comment:
@IT-Ideas perhaps we need to do this compute in the recursive way? If yes, we could store this field.
Because let's imagine this use case: 
- A depends on B
- B depends on C
- B.is_closed = True
- C.is_closed = False
So I think `A.is_blocked` should be True since C is not closed. 🤔 

What do you think? @RaoufGhrissi @IT-Ideas

Reply:
@xavierbol i think that it's more functional not technical, let's check with AJU if you want 
cc: @IT-Ideas

Reply:
Indeed, you are right. 🙂